src/core_processor/camera_calibration/calibration_diagnostics_visualizer.py

```diff
@@ -0,0 +1,152 @@
+import numpy as np
+import pyqtgraph as pg
+import pyqtgraph.opengl as gl
+from pyqtgraph.dockarea.Dock import Dock
+from pyqtgraph.dockarea.DockArea import DockArea
+from pyqtgraph.Qt import QtWidgets
+
+import cv2
+
+from src.core_processor.camera_calibration.calibration_dataclasses import LensDistortionCalibrationData
+
+
+class CalibrationDiagnosticsVisualizer:
```

Comment:
i have a lot of questions about this file, but ill come back to it later

Reply:
okay, coming back to this file:

I noticed that every single method is public and the init is rather large. Could you break apart the init into neat named functions?

Reply:
yes, definitely. 

This QT GUI thing is going to get moved out into the domain of the `session_manager` in the next refactor.

The giant `__init__` is a result of the fact that I didn't know what I was doing when I started, but now that I've figured out  how `pyqtgraph` works I'll be chopping the whole thing apart and basically rebuilding from scratch

Reply:
you dont have to break it apart in this PR
if there are plans to do that soon, im good.

it was big so i was like ahhhh *screams*